Extensions/PlatformBehavior/platformerobjectruntimebehavior.ts

```diff
@@ -1723,14 +1723,15 @@ namespace gdjs {
       behavior._requestedDeltaX +=
         this._floorPlatform!.owner.getX() - this._floorLastX;
       // See `beforeUpdatingObstacles` for the logic for the Y axis.
+
+      //Jumping
+      behavior._checkTransitionJumping();
```

Comment:
Can you explain in plain English why this change enabled to fix the issue? :)

Reply:
+1 I'm not sure I understand why this needs to happen in the beforeMovingX, and not in the checkTransitionBeforeY anymore?
Also, should it happen in the beforeMovingY() too?

Reply:
Sorry I meant to move it in `checkTransitionBeforeX` not `beforeMovingX`. I added a comment.

Reply:
> Also, should it happen in the beforeMovingY() too?

The check is about the jump key. It won't change during `doStepPreEvents`.